networkx/algorithms/connectivity/tests/test_kcomponents.py

```diff
@@ -126,34 +126,23 @@ def test_karate_component_number():
 
 
 def test_torrents_and_ferraro_detail_3_and_4():
-    solution = {
-        3: [{25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 42},
-            {44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 61},
-            {63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 79, 80},
-            {81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 93, 94, 95, 99, 100},
-            {39, 40, 41, 42, 43},
-            {58, 59, 60, 61, 62},
-            {76, 77, 78, 79, 80},
-            {96, 97, 98, 99, 100},
-        ],
-        4: [{35, 36, 37, 38, 42},
-            {39, 40, 41, 42, 43},
-            {54, 55, 56, 57, 61},
-            {58, 59, 60, 61, 62},
-            {73, 74, 75, 79, 80},
-            {76, 77, 78, 79, 80},
-            {93, 94, 95, 99, 100},
-            {96, 97, 98, 99, 100},
-        ],
-    }
     G = torrents_and_ferraro_graph()
     result = nx.k_components(G)
+    # In this example graph there are 8 3-components, 4 with 15 nodes
+    # and 4 with 5 nodes.
+    assert_true(len(result[3]) == 8)
```

Comment:
Same comment as above, these can all be `assert_equals`, and `assert_greater_equals` below.